hertzbeat-e2e/pom.xml

```diff
@@ -74,4 +74,16 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.testcontainers</groupId>
+                <artifactId>testcontainers-bom</artifactId>
+                <version>${testcontainers.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
 </project>
```

Comment:
You can fill in a blank line here..

![image](https://github.com/user-attachments/assets/51324fad-bb0d-413c-947a-c5c28071cbb3)